src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -4501,6 +4501,15 @@ def close(self) -> None:
         except Exception as e:
             logger.warning(f"Memory cleanup failed: {e}")
 
+        # LLM client cleanup
+        try:
+            if hasattr(self, 'llm') and self.llm:
+                llm_client = getattr(self.llm, '_client', None)
+                if llm_client and hasattr(llm_client, 'close'):
+                    llm_client.close()
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)